packages/block/src/types.ts

```diff
@@ -43,6 +43,17 @@ export interface BlockOptions {
    * difficulty takes precedence over a provided static `difficulty` value.
    */
   calcDifficultyFromHeader?: BlockHeader
+  /**
+   * A block object by default gets frozen along initialization. This gives you
```

Comment:
```suggestion
A block object by default gets frozen during initialization. This gives you
```
Just a tiny idiomatic change here...

This comment block is really helpful...makes the rationale for freezing very clear.

Reply:
Will leave this out since I don't want to dismiss the approval, hope that's ok.